std/container/array.d

```diff
@@ -544,6 +544,17 @@ if (!is(immutable T == immutable bool))
         return _data.refCountedStore.isInitialized ? _data._capacity : 0;
     }
 
+    /**  
+     * Returns: the internal representation of the array.
+     *
+     * Complexity: $(BIGOH 1).
+     */
+
+    T[] data() @system
```

Comment:
Unit test? Preferably one included in the documentation a la https://dlang.org/spec/unittest.html#documented-unittests